### ISSUE TITLE
Release 2.0

### ISSUE THREAD

**Issue by ekohl** [ralphbean/bugwarrior#1030]
The last release was done in 2020, but git commits are much more recent. Any plans to create a new release?

**Comment by ryneeverett** [comment]
Yes, we're close. There are two prerequisites, neither of which is blocking the other.

- **Technical**: Getting the code base ready for release is primarily my responsibility. I've marked [milestones](https://github.com/GothenburgBitFactory/bugwarrior/milestone/1) for the issues I think we should complete or at least investigate before releasing and assigned a couple to myself. If folks wanted to contribute to the unassigned issues to push this along it would most certainly be welcome.
- **Administrative**: This is out of my hands. We're still transitioning to @GothenburgBitFactory and have been having difficulties establishing communication channels. To this end:
    - @tbabej -- Ralph sent a couple emails to email addresses he believed were yours last week but it seems like they didn't get through. Is there a better way to contact you?

**Comment by JoshMock** [comment]
Just curious if there have been any updates on this. Arch Linux just bumped to Python 3.12, causing `bugwarrior-pull` to be broken due to missing the fix in https://github.com/GothenburgBitFactory/bugwarrior/pull/1019. I can run it in an env with an older Python build, so I have a workaround, but it'd be great to get all the latest changes out there. :black_heart:

**Comment by ryneeverett** [comment]
Not much of an update:

- Technical: I'm very occasionally making marginal progress towards the milestones.
- Administrative: The taskwarrior community is [planning to start having meetings](https://github.com/GothenburgBitFactory/taskwarrior/discussions/3403) soon and that should be a good opportunity to establish more reliable communication channels.

Re: Arch -- @JoshMock Am I correct in understanding that all Arch packages use system python which is now 3.12? I find it hard to believe anything works on Arch without more flexibility than that.

**Comment by JoshMock** [comment]
> Re: Arch -- @JoshMock Am I correct in understanding that all Arch packages use system python which is now 3.12? I find it hard to believe anything works on Arch without more flexibility than that.

Correct. The downside of a rolling update Linux distro. The upside is that we help package maintainers by finding all the bugs before everyone else upgrades. 😉

Anyway, Arch's recommended workaround is to use one of Python's many packaging tools instead, depending on your need. I used `pipx` to get Bugwarrior working again, so I'm good for now. 

Thanks for all your hard work maintaining this super valuable tool!

**Comment by ryneeverett** [comment]
Almost two years later I think we're finally ready. All the milestones are closed out, the changelog is updated, and the version has been bumped.

@ralphbean Could you please add at least one person in @GothenburgBitFactory as a pypi collaborator? (https://pypi.org/user/djmitche/, https://pypi.org/user/tbabej/, https://pypi.org/user/ryneeeverett/ *yes, I misspelled my handle on pypi* 🤦)

*Edit 11/23/25: I'm attempting to reach out to Ralph on matrix, but I'm not sure I have the right handle.*

*Edit 11/24/25: I sent a message to the couple known email addresses I have for Ralph.*

**Comment by ralphbean** [comment]
Ryne found me! Yes - I'll add all three people asap. Thank you so much for taking the project forwards!

**Comment by ryneeverett** [comment]
Thanks, just created the release on pypi and github!
